### PR TITLE
Show last four in the redacted digits

### PR DESCRIPTION
**Pull request by tanatorn**
Shows last four digits when using `hidden_card_number`
image

Optimizes one of the query because it made me sad
* before
```
  User Load (0.7ms)  SELECT "users".* FROM "users" WHERE "users"."id" = $1 LIMIT $2  [["id", 2658], ["LIMIT", 1]]
  ↳ app/views/events/card_overview.html.erb:147
  User Load (1.0ms)  SELECT "users".* FROM "users" WHERE "users"."id" = $1 LIMIT $2  [["id", 2596], ["LIMIT", 1]]
  ↳ app/views/events/card_overview.html.erb:147
  User Load (2.3ms)  SELECT "users".* FROM "users" WHERE "users"."id" = $1 LIMIT $2  [["id", 2385], ["LIMIT", 1]]
  ↳ app/views/events/card_overview.html.erb:147
  User Load (0.9ms)  SELECT "users".* FROM "users" WHERE "users"."id" = $1 LIMIT $2  [["id", 2372], ["LIMIT", 1]]
  ↳ app/views/events/card_overview.html.erb:147
  User Load (0.9ms)  SELECT "users".* FROM "users" WHERE "users"."id" = $1 LIMIT $2  [["id", 2046], ["LIMIT", 1]]
  ↳ app/views/events/card_overview.html.erb:147
  User Load (0.9ms)  SELECT "users".* FROM "users" WHERE "users"."id" = $1 LIMIT $2  [["id", 1803], ["LIMIT", 1]]
  ↳ app/views/events/card_overview.html.erb:147
  User Load (2.1ms)  SELECT "users".* FROM "users" WHERE "users"."id" = $1 LIMIT $2  [["id", 34], ["LIMIT", 1]]
  ↳ app/views/events/card_overview.html.erb:147
  User Load (0.6ms)  SELECT "users".* FROM "users" WHERE "users"."id" = $1 LIMIT $2  [["id", 892], ["LIMIT", 1]]
  ↳ app/views/events/card_overview.html.erb:147
  CACHE User Load (0.0ms)  SELECT "users".* FROM "users" WHERE "users"."id" = $1 LIMIT $2  [["id", 892], ["LIMIT", 1]]
  ↳ app/views/events/card_overview.html.erb:147
```
* after
```
  User Load (0.4ms)  SELECT "users".* FROM "users" WHERE "users"."id" IN ($1, $2, $3, $4, $5, $6, $7, $8)  [["id", 2658], ["id", 2596], ["id", 2385], ["id", 2372], ["id", 2046], ["id", 1803], ["id", 34], ["id", 892]]
  ↳ app/views/events/card_overview.html.erb:135
```

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -45,4 +45,7 @@ yarn-debug.log*
 # Ignore (and don't ignore) for dotenv.
 .env
 .env.me
-!.env.project
\ No newline at end of file
+!.env.project
+
+# IntelliJ
+.idea
```
Comment (tanatorn) [inline .gitignore]: adds intellij cause i use intellij

File: app/controllers/events_controller.rb [hunk 2]
```diff
@@ -124,7 +124,7 @@ def emburse_card_overview
 
   def card_overview
     @stripe_cards = @event.stripe_cards.includes(:stripe_cardholder, :user).order("created_at desc")
-    @stripe_cardholders = StripeCardholder.where(user_id: @event.users.pluck(:id)).order("created_at desc")
+    @stripe_cardholders = StripeCardholder.where(user_id: @event.users.pluck(:id)).includes(:user).order("created_at desc")
```
Comment (maxwofford) [inline app/controllers/events_controller.rb]: Thanks for catching this!

File: app/models/stripe_card.rb [hunk 3]
```diff
@@ -55,7 +55,7 @@ def formatted_card_number
   end
 
   def hidden_card_number
-    "•••• •••• •••• ••••"
+    "•••• •••• •••• #{last4}"
```
Comment (maxwofford) [inline app/models/stripe_card.rb]: That's great! Could we keep this hidden this in transparency mode? I'd like to see the last 4 digits when looking at a list of my teammember's cards, but I'd be sketched out if I saw these details when looking over someone else's account.

**Review by maxwofford — CHANGES_REQUESTED** [review]

**Comment by tanatorn** [comment]
When not signed in:
image

When signed in as an organizer:
image

**Comment by tanatorn** [comment]
PTAL @maxwofford

**Comment by maxwofford** [comment]
This looks good to go– thanks!

**Pull request merged**